sdk/appconfiguration/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/properties/AppConfigurationProperties.java

```diff
@@ -33,31 +29,13 @@ public final class AppConfigurationProperties {
      * Prefix for client configurations for connecting to configuration stores.
      */
     public static final String CONFIG_PREFIX = "spring.cloud.azure.appconfiguration";
-    
-    /**
-     * Separator for multiple labels.
-     */
-    public static final String LABEL_SEPARATOR = ",";
-
-    /**
-     * Context for loading configuration keys.
-     */
-    @NotEmpty
-    private String defaultContext = "application";
 
     private boolean enabled = true;
 
     private List<ConfigStore> stores = new ArrayList<>();
 
-    /**
-     * Alternative to Spring application name, if not configured, fallback to default Spring application name
-     **/
-    private String name;
-
-    @NestedConfigurationProperty
-    private AppConfigManagedIdentityProperties managedIdentity;
-
-    private boolean pushRefresh = true;
+    @Nullable
+    private String clientId; // Optional: client_id of the managed identity
```

Comment:
This `clientId` seems abrupt here, `spring.cloud.azure.appconfiguration.client-id`, what does this property mean?

Reply:
This is a historical artifact from when this library used REST, not the SDK. It was a way to provide a client id for auth. I think this is one of the things we might want to just remove. It is a 4.0 and we can make the breaking change.